pyt/__main__.py

```diff
@@ -300,9 +308,19 @@ def main(command_line_args=sys.argv[1:]):
         )
     )
     if args.json:
-        json.report(vulnerabilities, sys.stdout)
+        if args.baseline:
```

Comment:
Can the baseline file be normal text?  Like that:
```
2 vulnerabilities found:
Vulnerability 1:
File: /home/gunal/kelime_kok_ayirici/app.py
 > User input at line 98, trigger word "form[":
	 _ID = request.form['_id']
Reassigned in:
	File: /home/gunal/kelime_kok_ayirici/app.py
	 > Line 64: save_1__ID = _ID
	File: /home/gunal/kelime_kok_ayirici/app.py
	 > Line 104: temp_1__ID = _ID
	File: /home/gunal/kelime_kok_ayirici/db.py
	 > Line 64: _ID = temp_1__ID
	File: /home/gunal/kelime_kok_ayirici/app.py
	 > Line 64: _ID = _ID
File: /home/gunal/kelime_kok_ayirici/db.py
 > reaches line 80, trigger word "execute(":
	~call_4 = ret_cursor.execute(sql, (int(_IsTrue), str(_UserSuggestion), datetime.datetime.now(), int(_ID)))

Vulnerability 2:
File: /home/gunal/kelime_kok_ayirici/app.py
 > User input at line 99, trigger word "form[":
	 _IsTrue = request.form['_isTrue']
Reassigned in:
	File: /home/gunal/kelime_kok_ayirici/app.py
	 > Line 64: save_1__IsTrue = _IsTrue
	File: /home/gunal/kelime_kok_ayirici/app.py
	 > Line 104: temp_1__IsTrue = _IsTrue
	File: /home/gunal/kelime_kok_ayirici/db.py
	 > Line 64: _IsTrue = temp_1__IsTrue
	File: /home/gunal/kelime_kok_ayirici/app.py
	 > Line 64: _IsTrue = _IsTrue
File: /home/gunal/kelime_kok_ayirici/db.py
 > reaches line 80, trigger word "execute(":
	~call_4 = ret_cursor.execute(sql, (int(_IsTrue), str(_UserSuggestion), datetime.datetime.now(), int(_ID)))
```
if it can be we should parse it differently

Reply:
So the baseline file can only be JSON 👍

Reply:
Ok, i got it.